lib/node_runner.go

```diff
@@ -167,146 +202,36 @@ func (nr *NodeRunner) ConnectValidators() {
-	CheckNodeRunnerHandleBallotCheckResult,
-	CheckNodeRunnerHandleACCEPTBallotStore,
-}
-
-func (nr *NodeRunner) SetHandleMessageFromClientCheckerFuncs(
-	deferFunc sebakcommon.CheckerDeferFunc,
-	f ...sebakcommon.CheckerFunc,
-) {
-	if len(f) > 0 {
-		nr.handleMessageFromClientCheckerFuncs = f
-	}
-
-	if deferFunc == nil {
-		deferFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	nr.handleMessageFromClientCheckerDeferFunc = deferFunc
-}
-
-func (nr *NodeRunner) SetHandleBaseBallotCheckerFuncs(
-	deferFunc sebakcommon.CheckerDeferFunc,
-	finishedFunc sebakcommon.CheckerDeferFunc,
-	f ...sebakcommon.CheckerFunc,
-) {
-	if len(f) > 0 {
-		nr.handleBaseBallotCheckerFuncs = f
-	}
-
-	if deferFunc == nil {
-		deferFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	if finishedFunc == nil {
-		finishedFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	nr.handleBaseBallotCheckerDeferFunc = deferFunc
+func (nr *NodeRunner) SetHandleINITBallotCheckerFuncs(f ...sebakcommon.CheckerFunc) {
+	nr.handleINITBallotCheckerFuncs = f
 }
 
-func (nr *NodeRunner) SetHandleINITBallotCheckerFuncs(
-	deferFunc sebakcommon.CheckerDeferFunc,
-	finishedFunc sebakcommon.CheckerDeferFunc,
-	f ...sebakcommon.CheckerFunc,
-) {
-	if len(f) > 0 {
-		nr.handleINITBallotCheckerFuncs = f
-	}
-
-	if deferFunc == nil {
-		deferFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	if finishedFunc == nil {
-		finishedFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	nr.handleINITBallotCheckerDeferFunc = deferFunc
+func (nr *NodeRunner) SetHandleSIGNBallotCheckerFuncs(f ...sebakcommon.CheckerFunc) {
+	nr.handleSIGNBallotCheckerFuncs = f
 }
 
-func (nr *NodeRunner) SetHandleSIGNBallotCheckerFuncs(
-	deferFunc sebakcommon.CheckerDeferFunc,
-	finishedFunc sebakcommon.CheckerDeferFunc,
-	f ...sebakcommon.CheckerFunc,
-) {
-	if len(f) > 0 {
-		nr.handleSIGNBallotCheckerFuncs = f
-	}
-
-	if deferFunc == nil {
-		deferFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	if finishedFunc == nil {
-		finishedFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	nr.handleSIGNBallotCheckerDeferFunc = deferFunc
+func (nr *NodeRunner) SetHandleACCEPTBallotCheckerFuncs(f ...sebakcommon.CheckerFunc) {
+	nr.handleACCEPTBallotCheckerFuncs = f
 }
 
-func (nr *NodeRunner) SetHandleACCEPTBallotCheckerFuncs(
-	deferFunc sebakcommon.CheckerDeferFunc,
-	finishedFunc sebakcommon.CheckerDeferFunc,
-	f ...sebakcommon.CheckerFunc,
-) {
-	if len(f) > 0 {
-		nr.handleACCEPTBallotCheckerFuncs = f
-	}
-
-	if deferFunc == nil {
-		deferFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	if finishedFunc == nil {
-		finishedFunc = sebakcommon.DefaultDeferFunc
-	}
-
-	nr.handleACCEPTBallotCheckerDeferFunc = deferFunc
+func (nr *NodeRunner) SetHandleMessageCheckerDeferFunc(f sebakcommon.CheckerDeferFunc) {
+	nr.handleMessageCheckerDeferFunc = f
 }
 
 func (nr *NodeRunner) handleMessage() {
 	for message := range nr.network.ReceiveMessage() {
 		var err error
 
 		if message.IsEmpty() {
-			nr.log.Error("got empty message`")
+			nr.log.Debug("got empty message`")
```

Comment:
Could you explain why this was changed ?

Reply:
@Geod24 Oops, it should be `Error`.